runtime/tests/Test_LightningCoreQIS.cpp

```diff
@@ -1588,80 +1588,6 @@ TEST_CASE("Test __quantum__qis__QubitUnitary with num_qubits=2", "[CoreQIS]")
     __quantum__rt__finalize();
 }
 
-TEST_CASE("Test __rt__device registering a custom device with shots=500 and device=lightning",
```

Comment:
We're moving all `__rt__device` unit tests to `Test_LoadingThridParty.cpp`. right? I don't have any obligations but I may suggest renaming `Test_LoadingThridParty.cpp` to something more general, such as `Test_DLManager.cpp` or `Test_LoadingDrivers.cpp`. Also we have `Test_LightningDriver.cpp` that may be a good place for these tests as well.

Reply:
Renamed: https://github.com/PennyLaneAI/catalyst/pull/327/commits/3b2e82857d1df17dd6e58c594a1d3c23fe3dc14b